src/screens/ProfileScreen/elements/LinkAndSocialMedia.tsx

```diff
@@ -116,15 +120,21 @@ const styles = StyleSheet.create({
   },
   copyLinkContainer: {
     borderWidth: 1,
-    borderColor: COLORS.gray,
+    borderColor: COLORS.gray200,
     borderRadius: 8,
     justifyContent: 'center',
     alignItems: 'center',
     flexDirection: 'row',
     marginBottom: 10,
     width: '67%'
   },
-  copyLinkUrl: {padding: 10, flex: 1, fontWeight: '600', color: COLORS.balance_gray, fontSize: 12}
+  copyLinkUrl: {
+    padding: 10,
+    flex: 1,
+    fontFamily: fonts.inter[600],
+    color: COLORS.gray500,
+    fontSize: 12
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Several style properties have been updated, including borderColor, fontFamily, and color. These changes should enhance the visual appearance and readability of the component.
```diff
-    borderColor: COLORS.gray,
+    borderColor: COLORS.gray200,
+    justifyContent: 'center',
+    alignItems: 'center',
+    flexDirection: 'row',
+    marginBottom: 10,
+    width: '67%'
-    fontWeight: '600', 
-    color: COLORS.balance_gray, 
+    fontFamily: fonts.inter[600],
+    color: COLORS.gray500,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->